cmd/thanos/query.go

```diff
@@ -386,11 +388,25 @@ func runQuery(
 
 		router.Get("/-/healthy", func(w http.ResponseWriter, r *http.Request) {
 			w.WriteHeader(http.StatusOK)
-			if _, err := fmt.Fprintf(w, "Thanos Querier is Healthy.\n"); err != nil {
+			if _, err := fmt.Fprintf(w, "Thanos Querier is healthy.\n"); err != nil {
 				level.Error(logger).Log("msg", "Could not write health check response.")
```

Comment:
logs should be lowercase. I know it's not your mistake here, but can we fix it? (: